data/sql_updates/create_sched_e_aggregate_candidate.sql

```diff
@@ -9,8 +9,10 @@ select
     sum(exp_amt) as total,
     count(exp_amt) as count
 from sched_e
-where exp_amt is not null
-and (memo_cd != 'X' or memo_cd is null)
+where
+    exp_amt is not null and
+    filing_form in ('F3X', 'F5') and
```

Comment:
For F3X filers the IEs will be in the SCHED_E table.  For F5 filers the IEs will be in the FORM_57 table.  Does this help?  

select * from form_5 f, FORM_57 s where f.sub_id = s.link_id and f.rpt_tp not in ('24', '48') and f.rpt_yr = 2015;
--147 rows returned but doesn't deal with amendments but these behave like regular financial forms because we are excluding 24/48-hour reports.

select * from form_5 f, SCHED_E s where f.sub_id = s.link_id and f.rpt_tp not in ('24', '48') and f.rpt_yr = 2015;
--No rows returned (as expected)